cookbook/catalog_structure/overriding_the_orm_product_value.rst

```diff
@@ -269,4 +269,8 @@ Finally, check that your mapping override is correct by launching the following
 
     php app/console doctrine:mapping:info
 
-Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class.
+Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class:
+
+.. code-block:: bash
+
+    php app/console doctrine:schema:update --dump-sql --force
```

Comment:
I think commands should be always written in prod environment but I can see upper that it is not the case. What are you thinking about that?
It also needs a cache clear and apache2 restart on production environment